plugins/wasmedge_stablediffusion/sd_func.cpp

```diff
@@ -104,6 +108,34 @@ sd_image_t *readControlImage(Span<uint8_t> ControlImage,
   return ControlImg;
 }
 
+void upscalerModel(const char *UpscaleModelPath, uint32_t UpscaleRepeats,
+                  int32_t NThreads, uint32_t Wtype, uint32_t BatchCount, 
+                  sd_image_t *Results) {
+  int upscale_factor = 4;  // unused for RealESRGAN_x4plus_anime_6B.pth
+  upscaler_ctx_t* upscaler_ctx = new_upscaler_ctx(UpscaleModelPath, NThreads, static_cast<sd_type_t>(Wtype));
```

Comment:
It looks like there is a `new` without a delete. Considering to use `unique_ptr` instead.